articles/clients/index.md

```diff
@@ -4,186 +4,167 @@ description: Explains the basics of creating and using Auth0 Clients.
 
-![](/media/articles/applications/applications-logs-auditing.png)
+- **Allowed Origins (CORS)**: Set of URLs that will be allowed to make requests from JavaScript to Auth0 API (typically used with CORS). This prevents same-origin policy errors when using Auth0 from within a web browser. By default, all your callback URLs will be allowed. This field allows you to enter other origins if you need to. You can specify multiple valid URLs by comma-separating them. You can use the star symbol as a wildcard for subdomains (`*.google.com`). Notice that querystrings and hash information are not taking into account when validating these URLs.
 
-The logs include many of the actions performed by the user:
+- **JWT Expiration (seconds)**: The amount of time (in seconds) before the Auth0 [JWT](/jwt) expires. The default value is `36000`, which maps to 10 hours.
 
-* Logging in to a Client;
-* Failing to log in to a Client;
-* Signing up;
-* Requesting a password change;
+- **Use Auth0 instead of the IdP to do Single Sign On**: If enabled, this setting prevents Auth0 from redirecting authenticated users with valid sessions to the identity provider (such as Facebook, ADFS, and so on).
 
-You can download the event logs using the [Management API](/api/management/v2#!/Logs/get_logs) or view them via the [Management Dashboard](${manage_url}/#/logs)
-
-While the [Management Dashboard](${manage_url}/#/logs) displays the log data in a neatly formatted manner, clicking on the row corresponding to a particular event displays the raw data, which looks something like this:
+::: panel-info Auth0 Rules
+You can use [Rules](/rules) to further customize the authentication process for your client. Within the context of a rule, you have access to the client the user is authenticating to, which is useful if you want to apply coarse-grained authorization policies for your clients, for example:
+- Only HR officials can access Client X
+- Only US-based users can access Client Y
+For more details on rules and how to use them refer to: [Rules](/rules).
+:::
 
-```json
-{
-  "date": "2016-10-04T15:27:38.509Z",
-  "type": "f",
-  "description": "Invalid thumbprint (configured: 634AB4651FCA2F623563BE32EDA32DE565219118. calculated: BDEBFBFBA786C2D97F2125274793E32643358E81)",
-  "connection": "SSOCircle",
-  "connection_id": "con_T60D5poVozRmw77h",
-  "client_id": "UEsQCe4RVHYDSQ2zlLWIAHSqDhpsYyTG",
-  "client_name": "N/A",
-  "ip": "108.248.62.158",
-  "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36",
-  "strategy": "samlp",
-  "strategy_type": "enterprise"
-}
-```
+## Sample Clients
```

Comment:
would it make sense to remove these samples from here and direct the user to the [Architecture Scenarios doc](https://auth0.com/docs/architecture-scenarios)? that way we can focus the effort on maintaining scenarios in a single place. wdyt?